includes/donors/class-give-donor-wall.php

```diff
@@ -193,7 +194,7 @@ public function parse_atts( $atts ) {
 				'ids'               => '',
                 'cats'              => '',
                 'tags'              => '',
-				'columns'           => 'best-fit',
```

Comment:
@JoshuaHungDinh Can you explain what will happen if the saved shortcode has a `best-fit` value? Does shortcode output look fine?

Reply:
@ravinderk We no longer have a best-fit value as an option. I just tested it in a shortcode, having 'best-fit' is displaying the maximum amount of columns available.

Reply:
@JoshuaHungDinh Thank you for testing it.